content/en/docs/contribute/intermediate.md

```diff
@@ -324,38 +324,43 @@ graphical Git client instead.
 You only need to clone the repository once per physical system where you work
 on the Kubernetes documentation.
 
-1.  In a terminal window, use `git clone` to clone the repository. You do not
-    need any credentials to clone the repository.
+1.  Create a fork of the `kubernetes/website` repository on GitHub. In your
```

Comment:
The current page mentions “_If you have not done so,_ …”&mdash;you're intentionally removing that phrase, right?

Reply:
Yes. I thought it was redundant. Am I overstepping?

Reply:
Nope! Go for it, just wanted to check your intent.